extensions/kafka-client/runtime/pom.xml

```diff
@@ -45,7 +45,7 @@
 
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-caffeine</artifactId>
```

Comment:
Why did we have caffeine as dependency?

Reply:
It was used for caching Subjets in SubjectSubstitution but then dropped completely.